### PR TITLE
Use LXQtCompilerSettings

### PR DESCRIPTION
**Pull request by agaida**
fixes lxqt/lxqt/issues/1359

### CODE REVIEW

**Comment by agaida** [comment]
@palinek @tsujan @PCMan  - thoughts?

**Comment by tsujan** [comment]
I'm not good at cmake but it looks good.

**Comment by agaida** [comment]
it can wait a bit - no hurry

File: CMakeLists.txt [hunk 1]
```diff
@@ -70,6 +53,8 @@ add_definitions("-DLXQT_PANEL_VERSION=\"${LXQT_PANEL_VERSION}\"")
 
 include(LXQtPreventInSourceBuilds)
 include(LXQtTranslate)
+include(LXQtCompilerSettings)
+include(LXQtCompilerSettings NO_POLICY_SCOPE)
```
[the remark this answers is not in the document]
Comment (agaida) [reply CMakeLists.txt]: Good question - c&p it from an other package - maybe @luis-pereira can enlighten us.

[the remark this answers is not in the document]
Comment (agaida) [reply CMakeLists.txt]: nearly all other packages don't have it - i picked the only one who has this

**Pull request merged**